moped-etl/app/process/request.py

```diff
@@ -0,0 +1,48 @@
+#
+# Request Helper - Makes requests to a Hasura/GraphQL endpoint
+#
+
+import os
+import time
+import requests
+
+MAX_ATTEMPTS = int(os.getenv("HASURA_MAX_ATTEMPTS"))
+RETRY_WAIT_TIME = os.getenv("HASURA_RETRY_WAIT_TIME")
+HASURA_ENDPOINT = os.getenv("HASURA_ENDPOINT")
+HASURA_ADMIN_KEY = os.getenv("HASURA_ADMIN_KEY")
+
+def run_query(query):
+    """
+    Runs a GraphQL query against Hasura via an HTTP POST request.
+    :param query: string - The GraphQL query to execute (query, mutation, etc.)
+    :return: object - A Json dictionary directly from Hasura
+    """
+    # Build Header with Admin Secret
+    headers = {
+        "x-hasura-admin-secret": HASURA_ADMIN_KEY
+    }
+
+    # Try up to n times as defined by max_attempts
+    for current_attempt in range(MAX_ATTEMPTS):
+        # Try making the request via POST
+        try:
+            return requests.post(HASURA_ENDPOINT,
+                                 json={'query': query},
+                                 headers=headers).json()
+        except Exception as e:
+            print("Exception, could not insert: " + str(e))
```

Comment:
not wanting nitpick, but since I see you using two diff string formatting methods: imo [f-strings](https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python) are the best way to go these days because they're more readable.

Reply:
```python
  print(f"Exception, could not insert: {e}")
```

Reply:
I agree - the f-strings is much cleaner in Python, thanks for that.